File: javascript/1176.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split("\n");
var q = parseInt(lines.shift());
var soma = 0, prev = 0, curr = 0;

for (let i = 0; i < q; i++) {
    var val = parseInt(lines.shift());

    sum = 1;
    prev = 0;
    curr = 1;

    for (let m = 1; m <= val; m++) {
        sum = curr + prev;
        prev = curr;
        curr = sum;
    }
    console.log(`Fib(${val}) = ${prev}`);
}

File: C/1176.c
#include <stdio.h>
#include <stdlib.h>
#include <stdint.h>

int main() {
    int i, quant;
    unsigned long long curr = 1, prev = 0, sum = 1;

    scanf("%d", &quant);
    int values[quant];

    for (i = 0; i < quant; i++) {
        scanf("%d", &values[i]);
    }

    for (i = 0; i < quant; i++) {
        sum = 1;
        prev = 0;
        curr = 1;

        for (int j = 1; j <= values[i]; j++) {
            sum = curr + prev;
            prev = curr;
            curr = sum;
        }

        printf("Fib(%d) = %llu\n", values[i], prev);
    }
    return 0;
}